### ISSUE TITLE
Provide API aggregator

### ISSUE THREAD

**Issue by dwbrown2** [kubecost/cost-model#157]
Current users want access to aggregated costs (i.e. by namespace and namespace label) so that they can run their own analysis internally. Let's provide data in a similar format to what is exposed on the Kubecost Allocation view.

**Comment by dwbrown2** [comment]
Example output: 

```
{
cert-manager: {totalcost: 0.0008650852680044428, avgramalloc: 0.014454714457194012, avgramused: 0.014454714457194012, avgramreq: 0, ramcost: 0.0007147588495890299, …}
cost-model: {totalcost: 0.003974564337288486, avgramalloc: 0.03329478204250336, avgramused: 0, avgramreq: 0.03329478204250336, ramcost: 0.001617964287288487, …}
}
```

**Comment by dwbrown2** [comment]
v1 of this api was released in 1.44.1 release
